Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -284,6 +284,26 @@ public class Functions
     },
 ```
 
+By default, Lambda Annotations will update the CloudFormation template's [description](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-description-structure.html) 
+field to include the version of Lambda Annotations that was used to modify the template.
+```
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Transform": "AWS::Serverless-2016-10-31",
+  "Description": "An AWS Serverless Application. This template is partially managed by Amazon.Lambda.Annotations (v0.9.0.0).",
+   ...
+```
+
+This version in the template allows AWS to report on these templates to measure the usage of this framework. This aids investigations and outreach if we find a critical bug, 
```

Comment:
Per internal discussion tweaked slightly to replace "request level" with "CloudFormation stack" since this is unlike our usual useragent metrics.
> This description allow AWS to record the version and the usage of the Lambda Annotations framework in order to improve its quality. We record details at the CloudFormation stack level, and do not identify the application, library or tool that was deployed. Note that we do not record any personal information, such as usernames, email addresses or sensitive project-level information. 

Pushed up in an amended commit.